src/omero_zarr/cli.py

```diff
@@ -87,6 +87,20 @@ def _configure(self, parser):
             type=ProxyStringType("Image"),
             help="The Image from which to export Masks.",
         )
+        masks.add_argument(
+            "--group-rois",
```

Comment:
Perhaps "--split-masks" ?

Reply:
Could do, I'm confused by when you should use a ROI with multiple masks vs multiple ROIs with one mask in OMERO.

Reply:
All the shapes in one ROI are `UNION`'d, so multiple masks in one ROI is really only a storage optimization to prevent saving a bunches of `0`s.